### DISCUSSION TITLE
Cost of PoC for Full IPAM

### DISCUSSION THREAD

**Discussion by tonydean37** [Azure/ipam#82] [General]
Hi,

I was wondering if someone would be able to help in regards to a rough cost of running the full IPAM solution, as I have been requested to provide this before approval is given.

Thank you.

**Comment by DCMattyG** [comment]
Hey there @tonydean37, I've created an Azure Calculator cost estimation for the Full deployment of IPAM:

https://azure.com/e/3654eec59dd4419e83f7758b4bc3f0b5

With zero discount (Pay-As-You-Go), it's around ~$140/mo. This can vary depending on settings you tweak after the fact, if you decide to use a Private ACR, or with any discounting built into your companies EA agreement.

If you need help coming up with a more custom calculation based on your specific need, just let us know and we're happy to help out!

**Reply by tonydean37** [reply]
Good morning @DCMattyG,

This is great thank you :) I think it'll be enough to get the ball rolling, then we can always drill down into the costs in a bit more depth if needed.

Really appreciate your quick response on this one!

**Reply by serottez** [reply]
@DCMattyG Can you provide a customized monthly cost estimation for a deployment that uses the a.) Private Azure Container Registry and b) Azure Function + Azure storage account blob container for running the application container? Thank you in advance.

**Reply by DCMattyG** [reply]
Hey @serottez, I've answered in this in the other thread you have on this in #326
